s/babylon/camera/utils/on-camera-movement.ts

```diff
@@ -0,0 +1,18 @@
+import {TransformNode} from "@babylonjs/core/Meshes/transformNode.js"
+import {Vector3} from "@babylonjs/core/Maths/math.vector.js"
+import {v2} from "../../../utils/v2.js"
+
+export function onCameraMovement({
```

Comment:
this isn't a callback function, so let's remove the `on` prefix.

also, it's confusing what this actually does compared to `updateCameraMovement` exactly -- let's choose better names to distinguish these